frontend/js/src/explore/fresh-releases/components/ReleaseCardsGrid.tsx

```diff
@@ -62,9 +63,14 @@ export default function ReleaseCardsGrid(props: ReleaseCardReleaseProps) {
     return releaseKey;
   };
 
+  const mappedEntries =
+    direction === "descend"
+      ? Array.from(releaseMapping?.entries()).reverse()
+      : Array.from(releaseMapping?.entries());
+
```

Comment:
I'd say this is a tad more readable, but there's nothing wrong with your code
```suggestion
  const mappedEntries = Array.from(releaseMapping?.entries());
  if(direction === "descend") {
    mappedEntries.reverse()
  }
```